### PR TITLE
[FIX] Typos and clarifications

### PR DESCRIPTION
**Pull request by apjanke**
Just some suggested minor typo and style fixes.

### CODE REVIEW

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 1]
```diff
@@ -290,7 +290,7 @@ combined image rather than an image from each coil.
...
+| AcquisitionDuration               | RECOMMENDED. Duration (in seconds) of volume acquisition. Corresponds to DICOM Tag 0018,9073 `Acquisition Duration`. This field is REQUIRED for sequences that are described with the `VolumeTiming` field and that do not have the `SliceTiming` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `RepetitionTime`.                                                                                                                                                                                                                              |
```
Comment (apjanke) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]: changed "and that not have" to "and that do not have"

File: src/02-common-principles.md [hunk 2]
```diff
@@ -152,8 +152,8 @@ my_dataset/
     ...
 ```
 
-In this example **only the `rawdata` subfolder needs to be BIDS compliant**
-dataset. This specification does not prescribe anything about the contents of
+In this example **only the `rawdata` subfolder needs to a be BIDS compliant
```
Comment (effigies) [inline src/02-common-principles.md]:
```suggestion
In this example **only the `rawdata` subfolder needs to be a BIDS compliant
```

File: src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md [hunk 3]
```diff
@@ -290,7 +290,7 @@ combined image rather than an image from each coil.
...
+| AcquisitionDuration               | RECOMMENDED. Duration (in seconds) of volume acquisition. Corresponds to DICOM Tag 0018,9073 `Acquisition Duration`. This field is REQUIRED for sequences that are described with the `VolumeTiming` field and that do not have the `SliceTiming` field set to allowed for accurate calculation of "acquisition time". This field is mutually exclusive with `RepetitionTime`.                                                                                                                                                                                                                           |
```
Comment (effigies) [inline src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md]:
```suggestion
| AcquisitionDuration               | RECOMMENDED. Duration (in seconds) of volume acquisition. Corresponds to DICOM Tag 0018,9073 `Acquisition Duration`. This field is REQUIRED for sequences that are described with the `VolumeTiming` field and that do not have the `SliceTiming` field set to allow for accurate calculation of "acquisition time". This field is mutually exclusive with `RepetitionTime`.                                                                                                                                                                                                                             |
```

**Comment by effigies** [comment]
The easiest way to add yourself as a contributor is to edit the Wiki page (https://github.com/bids-standard/bids-specification/wiki/Contributors). This will be merged in before any release.

And to be clear, BIDS strongly encourages *any* level of engagement, so let me assure you that you have contributed enough to add yourself to the list.

**Comment by apjanke** [comment]
LOL. Fixes for the fixes pulled in. And I've added myself to the Contributors wiki page. Thanks!

**Comment by sappelhoff** [comment]
Thanks @apjanke!

**Pull request merged**